Quadriga/src/main/java/edu/asu/spring/quadriga/utilities/impl/NetworkUtils.java

```diff
@@ -0,0 +1,48 @@
+package edu.asu.spring.quadriga.utilities.impl;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Service;
+
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.service.network.IJsonCreator;
+import edu.asu.spring.quadriga.service.network.INetworkTransformationManager;
+import edu.asu.spring.quadriga.service.network.domain.ITransformedNetwork;
+import edu.asu.spring.quadriga.utilities.INetworkUtils;
+
+@Service
+public class NetworkUtils implements INetworkUtils {
+
+    @Autowired
+    private INetworkTransformationManager transformationManager;
+
+    @Autowired
+    private IJsonCreator jsonCreator;
+
+    private final Logger logger = LoggerFactory.getLogger(NetworkUtils.class);
+
+    @Override
+    public ResponseEntity<String> sendNetworkAsResponseEntity(String networkId) {
```

Comment:
I wouldn't even return a response entity but return the network in whatever requested format. leave creating the response entity to the controller.